src/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface State {
+  shipmentId: number;
+  toAddress: string;
+  fromAddress: string;
+  toZipCode: string;
+  fromZipCode: string;
+  weight: number;
+  marks: string[];
```

Comment:
Small note: according to the task, this field should be optional.

Reply:
done